bnpackage/delima.py

```diff
@@ -0,0 +1,139 @@
+            favorite_songs.append(self.name)
+            print(f"{self.name}'s added to favorites.\n")
+            
+            print("Favorite Songs:")
+            for song in favorite_songs:
+                print(f"- {song}")
+
+            input("\nPress enter to return.")
+
+        def remove_from_favorites(self): # Method 5
+            if self.name == "":
+                print("Please add a song name.")
+                return
+            
+            if self.name not in favorite_songs:
+                print(f"{self.name}'s not in favorites.\n")
+                
+            if self.name in favorite_songs:
+                favorite_songs.remove(self.name)
+                print(f"{self.name}'s removed from favorites.\n")
+
+            print("Favorite Songs:")
+            for song in favorite_songs:
+                print(f"- {song}")
+            
+            input("\nPress enter to return.")
+            
+        def menu(self):
+            choice = 0
+            while True:
+                print("\n- The Menu works like a playlist/player.")
+                print("You can list the songs you want")
+                print("and add it to your favorite songs.\n")
+
+                print("\n--------- Menu ----------")
+                print("[1] Display Song Details")
+                print("[2] Add Song Details")
+                print("[3] Clear Song Details")
+                print("[4] Add to favorites")
+                print("[5] Remove from favorites")
+                print("[6] Back to main menu")
+                print("-------------------------")
+
+                try :
+                    choice = int(input("Enter your choice: "))
+                except ValueError:
+                    print("\nYou need to input numbers (1-6).")
+
+                match choice:
+                    case 1: # Song Info
+                        os.system('cls')
+                        self.display_full_details()
+                        os.system('cls')
+                    case 2: # Add song info
+                        os.system('cls')
+                        self.add_song_details()
+                        os.system('cls')
+                    case 3: # Clear song info
+                        os.system('cls')
+                        self.clear_song_details()
+                        os.system('cls')
+                    case 4: # Add to favorites
+                        os.system('cls')
+                        self.add_to_favorites()
+                        os.system('cls')
+                    case 5: # Remove
+                        os.system('cls')
+                        self.remove_from_favorites()
+                        os.system('cls')
+                    case 6: # Return to main menu
+                        return
+                    case _:
+                        print("Invalid choice.")
+                        input("Please press enter to return.")
+                        os.system('cls')
```

Comment:
Please remove the os.system('cls) in case put the os.system('cls) at the top of match case and the last case. So, the code is to be clean to read.